draft-ietf-ace-oscore-profile.xml

```diff
@@ -455,7 +455,7 @@ A5                                      # map(5)
 
 
         <t>
-          If the client has requested an update to its access rights using the same OSCORE Security Context, which is valid and authorized, the AS MUST omit the 'cnf' parameter in the response, and MUST carry the client identifier and the context identifier (if it was set and included in the initial access token response by the AS) in the 'kid' field in the 'cnf' parameter of the token, with the same structure defined in <xref target="kid"/>. These identifiers need to be included in the response, in order for the RS to identify the previously generated Security Context.
+          If the client has requested an update to its access rights using the same OSCORE Security Context, which is valid and authorized, the AS MUST omit the 'cnf' parameter in the response, and MUST carry the client identifier and the context identifier (if it was set and included in the initial access token response by the AS) in the 'kid' field in the 'cnf' parameter of the token, with the same structure defined in <xref target="kid"/>. These identifiers need to be included in the token in order for the RS to identify the previously generated Security Context.
```

Comment:
Check this, but I'm pretty sure this has to be in the token, since the token is all the RS has available when it's determining whether this is an update or not.

Reply:
That's right. "in the response" was not incorrect (it needs to be in the response), but your change makes the sentence more precise. 👍